server/src/core/user/user.service.ts

```diff
@@ -67,19 +67,27 @@ export class UserService {
 
     assert(workspace, 'workspace is missing', BadRequestException);
 
+    const userSettings = await this.prismaService.client.userSettings.create({
+      data: { locale: 'en' },
+    });
+
+    const settings = { connect: { id: userSettings.id } };
```

Comment:
Connect `settings` to both `user` and `workspaceMember`, eventually to connect to only `workspaceMember`